### PR TITLE
update LangClass history.history miss acc, val_acc

### PR DESCRIPTION
**Pull request by yingkitw**
in def plot_history(history):
miss history.history['acc'] and history.history['val_acc']

### CODE REVIEW

**Comment by geert56** [comment]
What exactly is that you want to have changed? I cannot accept your request because it simply has too many changes in the notebook file. You use some other system and other versions of the software I assume. If it is just a change to accommodate some newer (or older) version of TensorFlow then I am sorry but we will not accept it. At some point all software is dated. If however you intend to suggest some improvement or have genuinely found a bug that please say so.

To summarize: if all you suggest is to change <code>history.history['accuracy']</code> and <code>history.history['val_accuracy']</code> to <code>history.history['acc'], history.history['val_acc']</code> then I will not accept the change since it will break in our own environment. You probably have a different version of Keras.

**Comment by geert56** [comment]
Close for lack of response.
